### PR TITLE
Add work experience page main structure

### PR DESCRIPTION
**Pull request by ShaneByrne0963**

### CODE REVIEW

**Review by stephendawsondev — APPROVED** [review]
LGTM!

**Comment by Dee-McG** [comment]
Going to need to resolve merge conflicts locally

File: profiles/templates/components/work-experience-form.html [hunk 1]
```diff
@@ -5,20 +5,65 @@
   <h1 class="block text-violet-700 text-2xl font-bold mb-2">Work Experience</h1>
   <p class="block text-gray-700 text-sm mb-4">Please enter your work experience</p>
   <form method="post">
-    {{ work_experience_form.organization|as_crispy_field }}
-    {{ work_experience_form.position|as_crispy_field }}
+    {% csrf_token %}
+    <div class="md:w-[50%]">
+      {{ work_experience_form.organization|as_crispy_field }}
```
Comment (Dee-McG) [inline profiles/templates/components/work-experience-form.html:10]: Does this properly crispy them? Shouldn't it just be |crispy

File: profiles/urls.py [hunk 2]
```diff
@@ -8,4 +8,5 @@
     path('update-summary/', views.UpdateSummary.as_view(), name='update-summary'),
     path('add-skill/<slug:skill>', views.AddSkill.as_view(), name='add_skill'),
     path('remove-skill/<slug:skill>', views.RemoveSkill.as_view(), name="remove_skill"),
+    path('add-responsibility', views.AddResponsibility.as_view(), name="add_responsibility")
```
Comment (Dee-McG) [inline profiles/urls.py]: Can we add trailing / to this and the above 2 URL's, just as best practice and to be consistent

**Review by Dee-McG — CHANGES_REQUESTED** [review]

File: profiles/templates/components/work-experience-form.html (hunk 1 above)
Comment (ShaneByrne0963) [reply profiles/templates/components/work-experience-form.html:10]: I am rendering the fields out individually so I can apply my own layout to them. I need to use as_crispy_field for this

Comment (Dee-McG) [reply profiles/templates/components/work-experience-form.html:10]: Oki

File: profiles/urls.py (hunk 2 above)
Comment (ShaneByrne0963) [reply profiles/urls.py]: Done

Comment (Dee-McG) [reply profiles/urls.py]: 9 & 10 too?? Sorry nit picking lol

Comment (ShaneByrne0963) [reply profiles/urls.py]: Ok that's done now

**Review by Dee-McG — APPROVED** [review]

**Pull request merged**